src/main/java/com/google/sps/servlets/RecommendationServlet.java

```diff
@@ -63,10 +72,20 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     db.close();
   }
 
-  private String getPath() {
-    ClassLoader classLoader = getClass().getClassLoader();
-    File file = new File(classLoader.getResource("META-INF/word2vec.db").getFile());
-    return file.toPath().toString();
+  private File getFile() {
+    Storage storage = StorageOptions.newBuilder().setProjectId("framecards").build().getService();
+    Blob blob = storage.get(BlobId.of("framecards.appspot.com", "word2vec.db"));
+
+    try {
+      File file = new File("/tmp/word2vec.db");
+      // Creates the file and downloads its content if it does not exist
```

Comment:
nit: creates new file object, not the .db file itself. Not sure this comment is necessary

Reply:
done.